pymetis/__init__.py

```diff
@@ -373,7 +373,7 @@ def part_graph(nparts, adjacency=None, xadj=None, adjncy=None,
                       eweights, options, recursive)
 
 
-def part_mesh(n_parts, connectivity, options=None, tpwgts=None, gtype=None):
+def part_mesh(n_parts, connectivity, options=None, tpwgts=None, gtype=None, ncommon=1):
```

Comment:
Please add a test that uses the parameter.

Reply:
Where should I add the test ?

Reply:
Somewhere here probably: https://github.com/inducer/pymetis/blob/main/test/test_partition_mesh.py